samples/app-link-unfurling-in-share-to-teams/csharp/assets/sample.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-link-unfurling-in-share-to-teams-csharp",
+    "source": "officeDev",
+    "title": "App link unfurling in share to teams",
+    "shortDescription": "Demonstrating feature of link unfurling in share to teams.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-link-unfurling-in-share-to-teams/csharp",
+    "longDescription": [
+      "Demonstrating feature of link unfurling in share to teams."
+    ],
+    "creationDateTime": "2022-12-14",
+    "updateDateTime": "2023-02-08",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "csharp"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "netframework"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "bot"
+      },
+	  {
+        "key": "TEAMS-FEATURES",
+        "value": "tab"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/app-link-unfurling-in-share-to-teams/csharp/LinkUnfurlingInShareToTeams/Images/app-link-unfurling-stt.gif",
+        "alt": "Solution UX showing app check in location"
```

Comment:
Wrong text:
Solution UX showing the feature of link unfurling in share to teams.

Reply:
changed